app/src/main/java/com/example/alinagavrila/employees_administration/Company.java

```diff
@@ -1,15 +1,143 @@
 package com.example.alinagavrila.employees_administration;
 
+import com.example.alinagavrila.employees_administration.constants.Enums.Platform;
+import com.example.alinagavrila.employees_administration.constants.Enums.ProgrammingLanguage;
 import com.example.alinagavrila.employees_administration.constants.Enums.Gender;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import javax.xml.namespace.NamespaceContext;
 
 public class Company {
     private static List<Employee> employeeList = new ArrayList<>();
 
-    public static void addNewEmployee(Employee employee) {
-        employeeList.add(employee);
+    public static Employee newEmployee() throws IOException {
+        Employee employee = null;
+        Gender gender;
+        BufferedReader bufferReader = new BufferedReader(new InputStreamReader(System.in));
+        String type = null;
+        System.out.print("New employee type: ");
+        try {
+            type = bufferReader.readLine();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        System.out.print("\nName:");
+        String name = bufferReader.readLine();
+        System.out.println("\nCNP:");
+        String CNP = bufferReader.readLine();
+        System.out.println("\nGender:");
+        String gen = bufferReader.readLine();
+        if(gen.equals("F")) {
+            gender = Gender.F;
+        }else {
+            gender = Gender.M;
+        }
+
+        switch(type) {
+            case "Administrative":
+                System.out.print("\nApproves Leave: ");
+                boolean approveLeaves = Boolean.parseBoolean(bufferReader.readLine());
```

Comment:
typo "approves"

Reply:
also, what happens if the user is not providing a boolean value? (the input is not true/false, yes/no)